docker.go

```diff
@@ -79,16 +94,36 @@ func (container *Container) Volume(hostPath, containerPath, mode string) *Contai
 func NewDockerContainer() *Container {
 	container := new(Container)
 	container.Type = "DOCKER"
-	container.Docker = &Docker{
-		Image:        "",
-		Network:      "BRIDGE",
-		PortMappings: make([]*PortMapping, 0),
-		Parameters:   make([]*Parameters, 0),
-	}
-	container.Volumes = make([]*Volume, 0)
+	container.EmptyVolumes()
+	container.Docker = &Docker{}
+	container.Docker.Container("").
+		Bridged().
+		SetPrivileged(false).
```

Comment:
yeah ... I see your point. So: the only thing would be to initialize docker?

```go
func NewDockerContainer() *Container {
	container := &Container{}
	container.Type = "DOCKER"
	container.Docker = &Docker{}

	return container
}
```
That's what I have done now as initializing with `network: "BRIDGE"` would be inconsistent then.
Only gotcha here is that apart from this PR being a breaking change it would be a different behavior as well. But better now than later.